docs/manual/document-format-reference/job-yaml-v12.md

```diff
@@ -833,6 +845,9 @@ Defines a notification for the job. You can include any of `onsuccess`, `onfailu
 
 `onsuccess`/`onfailure`/`onstart`/`onavgduration`/`onretryablefailure`
 
+::: Note: `onavgduration` also requires the following attribute set at the same level as `notification`
```

Comment:
```suggestion
::: tip
`onavgduration` also requires the following attribute set at the same level as `notification`
:::
```

Reply:
This is the tip box format:
![image](https://user-images.githubusercontent.com/271965/79531257-68eadb00-8037-11ea-8846-d469490cd876.png)